_data/tutorials.yaml

```
@@ -165,7 +165,7 @@ finding-distinct:
   problem: "you have events in a Kafka topic, and you want to filter out duplicate events based on a field in the event, producing a new stream of only unique events"
```

@riferrei I think @blueedgenick's suggestion above is worth heeding for both KStreams and KSQL flavors. I suggested a slight rewording of the shared problem statement. To me the introduction seems okay as is. What do you all think?
```suggestion
  problem: "you have events in a Kafka topic, and you want to filter out duplicate events based on a field in the event, producing a new stream of unique events per time window"
```

---

Yup, make sense!